disconnected/about.adoc

```diff
@@ -0,0 +1,61 @@
+
+A disconnected environment is an environment that does not have full access to the internet.
+
+{product-title} is designed to perform many automatic functions that depend on an internet connection, such as retrieving release images from a registry or retrieving update paths and recommendations for the cluster.
+Without a direct internet connection, you must perform additional setup and configuration for your cluster to maintain full functionality in the disconnected environment.
+
+[id="glossary-disconnected_{context}"]
+== Glossary of disconnected environment terms
+
+Although it is used throughout the {product-title} documentation, _disconnected environment_ is a broad term that can refer to environments with various levels of internet connectivity.
+Other terms are sometimes used to refer to a specific level of internet connectivity, and these environments might require additional unique configurations.
+
+The following table describes the different terms used to refer to environments without a full internet connection:
+
+.Disconnected environment terms
+[cols=".^2,.^4",options="header"]
+|====
+|Term |Description
+
+|Air-gapped network
+|An environment or network that is completely isolated from an external network.
+
+This isolation depends on a physical separation, or an "air gap", between machines on the internal network and any other part of an external network.
+Air-gapped environments are often used in industries with strict security or regulatory requirements.
+
+|Disconnected environment
+|An environment or network that has some level of isolation from an external network.
+
+This isolation could be enabled by physical or logical separation between machines on the internal network and an external network.
+Regardless of the level of isolation from the external network, a cluster in a disconnected environment does not have access to public services hosted by Red{nbsp}Hat and requires additional setup to maintain full cluster functionality.
+
+|Restricted Network
+|An environment or network with limited connection to an external network.
+
+A physical connection may exist between machines on the internal network and an external network, but network traffic is limited by additional configurations, such as with firewalls and proxies.
+|====
+
+[id="preferred-methods_{context}"]
+== Preferred methods for working with disconnected environments
+
+// Not sure about the heading above, I'm trying to find some phrasing that basically says "You should probably use these things if you don't have a strong reason to go with other options".
+
+You can choose between multiple options for most aspects of managing a cluster in a disconnected environment.
+For example, when mirroring images you can choose between using the oc-mirror {oc-first} plugin or using the `oc adm` command.
+
+However, some options provide a simpler and more convenient user experience for disconnected environments, and are the preferred method over their alternatives.
```

Comment:
```suggestion
However, some options provide a less complex user experience for disconnected environments which are also supported by Red Hat Technical Support.
```

Reply:
ONLY a suggestion. I'm kinda pushing the envelope here, but I think it gets you where you want to be with this - you need something stronger than "preferred" and I agree. WHO "prefers" is the question I think a customer might have, and "why do I care"? I think you need to make the customer care enough to follow the recommendation.

Reply:
@skopacz1 I've finished peer review. Highly polished work sir, as always, my compliments to the chef. :+1:

Reply:
I weighed in on your need to improve over the term "recommended". I agree with your desire to do so and provided a suggestion for your consideration. Otherwise, LGTM sir.

Reply:
BTW: vale flagged the word "may", and asked us to change it to "might", but I disagree and believe it is appropriate in the way you have used it.

Reply:
This overall section is tricky because some SSG rule tells us to avoid using "recommended" and to just tell the user to do the preferred thing. But in this case, there are legitimate use cases for the "less recommended" methods and we can't explicitly steer them away from those.

I see what your suggestion does about calling it less complex and that might be a good choice. That said, in this context I'm not worried about mentioning the supportability since I feel like that's already implied by the inclusion of these methods in the docs + suggesting them explicitly.

I'll think about these suggestions and see if I should include them in this doc, thanks!